public/static/locales/en/common.json

```diff
@@ -511,5 +511,6 @@
   "delete_event_type": "Delete Event Type",
   "confirm_delete_event_type": "Yes, delete event type",
   "integrations": "Integrations",
-  "settings": "Settings"
+  "settings": "Settings",
```

Comment:
I hope that this is all I need to add a new string. If there's something I'm missing, please let me know.